sqldelight-runtime/src/commonMain/kotlin/com/squareup/sqldelight/logs/LogSqliteDriver.kt

```diff
@@ -0,0 +1,80 @@
+package com.squareup.sqldelight.logs
```

Comment:
also needs licensing at top. Can just copy/paste from any other file

Reply:
fix in the new pr